typescript/phoenix-sdk/src/utils/connection.ts

```diff
@@ -0,0 +1,125 @@
+import { Connection, PublicKey, SYSVAR_CLOCK_PUBKEY } from "@solana/web3.js";
+import { ZSTDDecoder } from "zstddec";
+
+// Largest configured account size (multiplied by 2)
+const MAX_ACCOUNT_SIZE_BYTES = 1723488 * 2;
+
+/**
+ * Get market account data and clock data with zstd compression
+ * @param marketKey The market address
+ * @returns
+ */
+export const getConfirmedMarketsAndClockAccounts = async (
+  connection: Connection,
+  marketKeys: PublicKey[]
+): Promise<Buffer[]> => {
+  const rawAccounts = (
+    await connection.getMultipleAccountsInfo(
+      [...marketKeys, SYSVAR_CLOCK_PUBKEY],
+      "confirmed"
+    )
+  ).map((account) => account?.data);
+  if (!rawAccounts.every((account) => account !== undefined)) {
```

Comment:
easier to read as:

`if (rawAccounts.some((account) => account === undefined))`

Reply:
ah i was looking for the `any` function ty